src/plugin/customParseFormat/index.js

```diff
@@ -80,7 +81,31 @@ function correctHours(time) {
   }
 }
 
-function makeParser(format) {
+function makeParser(format, instance) {
+  const expressions = {
+    ...baseExpressions,
+    MMM: [matchWord, function (input) {
+      const locale = instance.$locale()
+      const { months, monthsShort } = locale
+      const matchIndex = monthsShort
+        ? monthsShort.findIndex(month => month === input)
+        : months.findIndex(month => month.substr(0, 3) === input)
+      if (matchIndex < 0) {
+        throw new Error(`Failed to parse "${input}" as MMM`)
```

Comment:
no error, just skip it like others

Reply:
done

Reply:
But suddenly it broke the tests on "Invalid Date" after parse corrupted string

Reply:
So, am I should to bring errors back?